File: Python/array_reverse.py
#Reverse an Array
arr = list(map(int, input("Enter array elements: ").split()))
start, end = 0, len(arr)-1

while start < end:
    arr[start], arr[end] = arr[end], arr[start]
    start += 1
    end -= 1

print("Reversed array:", arr)

File: CPP/array_reverse.cpp
#include<iostream>
using namespace std;

int main()
{
    int arr[5] = {2,3,4,5,6};
    int start = 0;          // index
    int end = 4;            // last index (size-1)
    int temp;

    while (start < end) {
        temp = arr[start];
        arr[start] = arr[end];
        arr[end] = temp;

        start++;
        end--;
    }

    cout << "The reverse order is: ";
    for(int i = 0; i < 5; i++) {
        cout << arr[i] << " ";
    }
    cout << endl;

    return 0;
}
